README.Rmd

```diff
@@ -0,0 +1,190 @@
+In this document you can enter in key details about yourself and your dissertation
+between the \-\-\- and \-\-\-.
+Note that any text after hashtags, \#, are comments only and are not computed. 
+
+The \#metadata\# section should be fairly self-explanatory.
+The \#front matter\# section includes pointers to various front matter elements,
+such as dedications, acknowledgements (thank you), abbreviations, and your abstract.
+Note that it points to files in a folder created by the template 
+called "front-and-back-matter/".
+If you look in this folder, you will find placeholders for your own
+abstract and acknowledgements.
+You can simply edit these and the 
+edited versions will be used when the thesis is compiled.
+This section also includes various options for 
+toggling lists of figures, tables, etc, in the final version.
+
+The \#bibliography\# section points to where the bibtex file (.bib) with
+all your bibliographic information resides.
+You can also just call your .bib file "references.bib" and replace the
+template version in the bib/ folder.
+Lastly, the \#formatting\# section offers options for changing the format
+ready for printing.
+You can mostly ignore the \#output\# section, for now.
+
+Below the second \-\-\- is some example text that will become 
+the preface of the thesis when it is compiled.
+So where is the real content of the dissertation?
+
+Just as the abstract and acknowledgements, etc,
+were kept in a "front-matter/" folder,
+so too are the individual chapters of your dissertation
+kept in a "chapters/" folder.
+The placeholder text is designed to take you through the
+basic syntax and more complicated options available when
+using _RMarkdown_,
+but once you feel comfortable and have created your own
+examples, feel free to delete the placeholder text
+(or even the files) and create your own in its place.
+
+_Markdown_ is easy to read as well as write,
+so there's no need for endless compilation.
+But when it comes time to compile the document,
+just turn to your main .Rmd file ('Untitled.Rmd' in our case)
+and either click the `Knit` button at the top
+or use the keyboard shortcut `Shift-Cmd-K` (Mac) or `Shift-Ctrl-K` (Windows).
+
+<!-- 
+## Knitting individual chapters
+
+To knit an individual chapter without compiling the entire thesis:
+1. open the **.Rmd** file of a chapter
+2. add a YAML header specifying the output format(s) (e.g. `bookdown::word_document2` for a word document you might want to upload to Google Docs for feedback from collaborators)
+3. Click the `knit` button (the output file is then saved in the root folder)
+
+As shown in the sample chapters' YAML headers, to output a single chapter to PDF, use:
+
+```yaml
+output:
+  bookdown::pdf_document2:
+    template: templates/iheiddiss.tex
+```
+This will format the chapter in the myiheidtex style but without including the front matter (table of contents, abstract, etc)
+-->
```

Comment:
Do we want to add this section back in?

Reply:
I can add it back in since it seems informative. However, the README is already quite long... What about we put it in the vignette instead with the next patch?